src/SupportMessage/SupportMessage.stories.tsx

```diff
@@ -21,4 +23,17 @@ const supportMessageArgs: SupportMessageProps = {
 
 Default.args = supportMessageArgs
 
+export const WithCustomDescription = Template.bind({})
+
+WithCustomDescription.args = {
+  type: 'info',
+  title: 'A SupportMessage using the Text component',
+  description: (
+    <Text tag="p">
+      Some text rendered using the Text component and a{' '}
+      <Link href={''}>Link</Link>
+    </Text>
```

Comment:
The story doesn't look right, I think the `Text` here is messing with the styling
```suggestion
    <>
      Some text rendered using the Text component and a{' '}
      <Link href={''}>Link</Link>
    </>
```

The following should fix the style